setup.cfg

```diff
@@ -34,8 +34,10 @@ universal = 1
 [flake8]
 max-line-length = 99
 exclude = docs, .git, __pycache__, .ipynb_checkpoints
-ignore = W503
-extend-ignore = SFS3  # String literal formatting using f-string.
+extend_ignore = W503,
+	N803,  # argument name 'X' should be lowercase 
+	N806,  # variable 'X' in function should be lowercase
+  SFS3  # String literal formatting using f-string.
```

Comment:
I'll fix the formatting the indentation here in the base PR, after this is merged.